src/runtime/reference.ts

```diff
@@ -25,30 +25,37 @@ export type SerializedReference = {
   entityStorageKey: string;
   // TODO(#4861): creationTimestamp shouldn't be optional.
   creationTimestamp?: string;
-  // TODO(mmandlis): add expiration time here as well.
+  expirationTimestamp?: string;
```

Comment:
can these be Dates as well? were you planning on changing these later?

Reply:
Date isn't serializable, but changed to number instead.